Editor/Scenes/ControlPanel/ControlPanel.gd

```diff
@@ -1,31 +1,30 @@
 extends Control
 
+signal play()
```

Comment:
do signals need parenthesis?

Reply:
Tends to be the way I write them usually, since you need an argument list if you have signals with arguments :)

Reply:
signals with arguments are passed through emit signals right?

Reply:
Yup, but specifying them here in the signal lets Godot auto-generate the right handler for it, arguments and all :D

Reply:
nice